### DISCUSSION TITLE
Modify API path OR mapping [Aug 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3215] [Discord Threads]
> 🙋‍♂️ **ghost**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@361928900588077057> Not possible directly on NocoDB. could use Nginx?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1006403977379848203)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
